build_tools/benchmarks/common/benchmark_driver.py

```diff
@@ -242,3 +245,25 @@ def __get_available_drivers_and_loaders(
       print(f"Available loaders: {available_loaders_str}")
 
     return available_drivers, available_loaders
+
+
+class IreeBenchmarkDriver(BenchmarkDriver):
+
+  def _parse_and_serialize_benchmark_metrics(
+      self, results_filename: pathlib.Path,
+      benchmark_stdout: str) -> BenchmarkMetrics:
+    iree_benchmark_json = json.loads(benchmark_stdout)
+    real_times = dict(unit="ns")
+    cpu_times = dict(unit="ns")
+    for metric in ["mean", "median", "stddev"]:
+      real_times[metric], cpu_times[metric] = get_google_benchmark_times(
+          iree_benchmark_json, metric)
```

Comment:
Instead of composing the dict object and converting it into `BenchmarkLatency` here. It seems like we can just let `get_google_benchmark_times` to parse all time metrics and return two `BenchmarkLatency`?